nengo_viz/static/viz_slider.js

```diff
@@ -9,6 +9,11 @@ VIZ.Slider = function(parent, args) {
     VIZ.Component.call(this, parent, args);
     var self = this;
 
+    //Check if user is filling in a number into a slider
+    self.filling_slider_value = false;
+
+    self.notify_msgs = [];
+
```

Comment:
In all of our other code, this is done with ```this``` rather than ```self```.  We've been using ```self``` only when needed (inside functions that are defined inside one of this class methods).